rbac/management/group/view.py

```diff
@@ -579,3 +568,43 @@ def validate_and_get_exclude_key(self, params):
                 exclude)
             raise serializers.ValidationError({key: _(message)})
         return exclude
+
+    def obtain_principals(self, request, group):
+        """Obtain principals based on request, supports exclusion."""
+        principals_from_params = self.filtered_principals(group, request)
+
+        exclude = self.validate_and_get_exclude_key(request.query_params)
+
+        if exclude == 'false':
+            page = self.paginate_queryset(principals_from_params)
+            serializer = PrincipalSerializer(page, many=True)
+        else:
+            # Get all the principals to be excluded
+            serializer = PrincipalSerializer(principals_from_params, many=True)
```

Comment:
Does this mean that we won't be paginating the results on our end, either?

Reply:
Right, we have to exclude all of them